website/theme.config.tsx

```diff
@@ -2,9 +2,33 @@
 /* eslint sort-keys: error */
 import { useRouter } from 'next/router';
 import { defineConfig, Giscus, useTheme } from '@theguild/components';
+import { MendableSearchBar } from '@mendable/search';
 
 export default defineConfig({
   docsRepositoryBase: 'https://github.com/dotansimha/graphql-code-generator/tree/master/website',
+  search: {
+    component: () => {
+      return (
+        <div className="hidden w-[250px] sm:block">
+          <MendableSearchBar
```

Comment:
the center or most of the search bar doesn't open the dialog on click, and it only happen on Firefox, it's fine with Chrome

![image](https://github.com/dotansimha/graphql-code-generator/assets/72823042/ae2ff890-e413-4ddc-bf41-66ce1d69308b)